src/pages/AppDetails/AppDetailsPage.tsx

```diff
@@ -98,7 +99,7 @@ class AppDetails extends React.Component<RouteComponentProps<AppId>, AppDetailsS
       version: ''
     };
     const restParams = {
-      duration: `${TrafficDetails.defaultDuration}s`,
+      duration: `${MetricsHelper.initDuration({}).duration!}s`,
```

Comment:
It is a bit weird to use `MetricsHelper.initDuration({}).duration!` here, as this function is really useful to compute some metrics related stuff such as the rate interval that depends on duration, which is of no use here. I think the only piece of code that is interesting to re-use is `MetricsDuration.initialDuration()` that takes duration from URL and/or session storage.

Unless there's something I didn't see, I think it's better to change, to have less coupling with the Metrics-specific stuff.

Reply:
You are right. I fixed all references.